src/components/@helpers/dropdown/Dropdown.tsx

```diff
@@ -30,7 +30,7 @@ export const Dropdown: React.FunctionComponent<DropdownProps> = ({}) => {
           type="button"
           onClick={() => setIsOpen(!isOpen)}
         >
-          -- Select issue template --
+          {buttonTitle}
```

Comment:
Title should change back to initial text, only when submit to api is successful - not on submit button click @KBandipo